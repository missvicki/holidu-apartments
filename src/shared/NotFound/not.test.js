import React from 'react';
import { shallow } from 'enzyme';
import NotFound  from './404.jsx';

describe('Not Found', () => {
  it('should not have regressed', () => {
    const component = shallow(<NotFound />);

    expect(component).toMatchSnapshot();
  });
});
