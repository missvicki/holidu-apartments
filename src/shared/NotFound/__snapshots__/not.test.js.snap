// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found should not have regressed 1`] = `
<Fragment>
  <Header />
  <div
    className="not_found__container"
  >
    <h1
      data-testid="404-text"
    >
      404 - Not Found
    </h1>
    <p>
      Sorry, the page you are looking for does not exist.
    </p>
    <p>
      You can always go back to the
      <Link
        to="/"
      >
        homepage
      </Link>
    </p>
  </div>
  <Footer />
</Fragment>
`;
